backend/prisma/schema.prisma

```diff
@@ -77,6 +77,7 @@ model Post {
   title      String
   body       String
   status     PostStatus   @default(DRAFT)
+  // @Lanny ⬇️
   images     Image[]
```

Comment:
*Part 1
@Lanny so this column doesn't seem to show up in the DB when I view it and we're also unable to query it. Do we need to annotate the relation here? We shouldn't need to right because we name it on the `Post` model... I think I might be a wee bit confused here. See *Part 2 below